### PR TITLE
add korea

### PR DESCRIPTION
**Pull request by phamleduy04**
I added Korea but idk why I when I tried to get `/v3/covid-19/gov` the array returns empty string. 
Screen Shot 2020-10-02 at 5 39 48 PM
 I tried to get `/v3/covid-19/gov/%20/` and it returns the request.

### CODE REVIEW

**Comment by ebwinters** [comment]
What is this line returning for Korea? My guess is Korea is not a country - it is either South or North
`const standardizedCountryName = nameUtils.getCountryData(country.trim()).country;`

**Comment by phamleduy04** [comment]
Its South Korea

**Comment by ebwinters** [comment]
Tests still failing m8 once they pass I'll give it a review

**Comment by phamleduy04** [comment]
Screen Shot 2020-10-04 at 11 10 52 PM
Screen Shot 2020-10-04 at 11 11 31 PM

Mine v2 and v3 are ok, something else breaking the test! @ebwinters

File: scrapers/covid-19/govScrapers/getGovData.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ const southAfricaData = require('./getSouthAfrica');
 const ukData = require('./getUK');
 const israelData = require('./getIsrael');
 const vietnamData = require('./getVietnam');
+const sKoreaData = require('./getSKorea');
```
Comment (ebwinters) [inline scrapers/covid-19/govScrapers/getGovData.js:13]: southKoreaData

File: scrapers/covid-19/govScrapers/getSKorea.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const axios = require('axios');
```
Comment (ebwinters) [inline scrapers/covid-19/govScrapers/getSKorea.js:1]: Call this file getSouthKorea

File: scrapers/covid-19/govScrapers/getSKorea.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	return city;
+};
+
+const sKoreaData = async () => {
+	try {
+		const res = await axios.default({ method: 'GET', url: 'http://ncov.mohw.go.kr/en/bdBoardList.do?brdId=16&brdGubun=162&dataGubun=&ncvContSeq=&contSeq=&board_id=&gubun=' });
+		const html = cheerio.load(res.data);
+		return html('table').children('tbody:first-of-type').children('tr').map(mapRows).get().filter(el => el.city !== 'Total');
+	} catch (err) {
+		logger.err('Error: Requesting Korea Gov Data failed!', err);
```
Comment (ebwinters) [inline scrapers/covid-19/govScrapers/getSKorea.js:33]: South

**Review by ebwinters — CHANGES_REQUESTED** [review]

**Review by ebwinters — APPROVED** [review]

**Pull request merged**